pom.xml

```diff
@@ -80,7 +80,7 @@
         <spotbugs.version>4.7.0</spotbugs.version>
         <sonatype.nexus.staging>1.6.3</sonatype.nexus.staging>
 
-        <kafka.version>3.2.1</kafka.version>
+        <kafka.version>3.2.3</kafka.version>
         <jackson.version>2.12.6</jackson.version>
         <jackson.databind.version>2.12.6.1</jackson.databind.version>
```

Comment:
Kafka 3.2.3 and 3.3.0 uses JAckson 2.13 IIRC. Would it make sense to align on that?

Reply:
Missed the updates. Will update.